src/components/Profile/components/Identification/index.js

```diff
@@ -648,7 +648,7 @@ const Identification = ({ profile, myProf, network, createSnackbar }) => {
                       alt="Profile"
                     />
 
-                    {network === 'online' && myProf && (
+                    {isOnline === 'online' && myProf && (
```

Comment:
Since isOnline is a boolean, you no longer have to compare it to a string value `"online"`
```suggestion
                    {isOnline && myProf && (
```

Reply:
You are right, I missed this one.